tgui/packages/tgui/components/goonstation/SectionWithTitleOverflow.js

```diff
@@ -0,0 +1,19 @@
+import { classes } from 'common/react';
+import { computeBoxClassName, computeBoxProps } from '../Box';
+import { Section } from '../Section';
+
+export const SectionWithTitleOverflow = props => {
+  const {
+    className,
+    ...rest
+  } = props;
+  return (
+    <Section
+      className={classes([
+        'SectionWithTitleOverflow',
+        className,
+        computeBoxClassName(rest),
+      ])}
+      {...computeBoxProps(rest)} />
```

Comment:
```suggestion
      ])}
      {...rest} />
```
The `Section` component should handle the rest, I think?